src/sentry/static/sentry/app/components/events/contexts/state.tsx

```diff
@@ -20,23 +21,40 @@ type Props = {
 };
 
 class StateContextType extends React.Component<Props> {
+  getStateTitle(name: string, type?: string) {
+    return name + (type ? ` (${upperFirst(type)})` : '');
+  }
+
   getKnownData(): KeyValueListData[] {
     const primaryState = this.props.data.state;
 
     return [
       {
         key: 'state',
-        subject:
-          t('State') + (primaryState.type ? ` (${upperFirst(primaryState.type)})` : ''),
+        subject: this.getStateTitle(t('State'), primaryState.type),
         value: primaryState.value,
       },
     ];
   }
 
+  getUnknownData(): KeyValueListData[] {
+    const {data} = this.props;
+
+    return Object.entries(data)
```

Comment:
I'm wondering if the `getUnknownData()` utility function you made could be re-used here?

Maybe add an optional `exclude` array of strings as a 3rd parameter.

```
getUnknownData(
  allData: Record<string, any>,
  knownKeys: string[],
  exclude?: string[],
)
```

Reply:
yeah... I believe that also `get?KnownData()` could be a reusable func.  I always wanted to do that before but never had time

Reply:
State context is in a slightly different format and is still a subject of discussion with the SDK team. I would like to keep its own method, at least for now.